src/main/java/frc/robot/Robot.java

```diff
@@ -27,6 +27,9 @@ public void robotInit() {
 
   @Override
   public void robotPeriodic() {
+    if (lift.getBottomSwitch()) {
+      lift.resetEncoderHeight();
+    }
   }
```

Comment:
if (lift.getBottomSwitch()) {
      lift.resetEncoderHeight();
    }

TO

if (lift.getBottomSwitch()) lift.resetEncoderHeight();